server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionEnvironmentAdapter.java

```diff
@@ -72,4 +72,9 @@ public <T extends CloudConnectionConfig> T adaptConfig(T connectionConfig) {
         return connectionConfig;
     }
 
+    @Override
+    public <T extends CloudConnectionConfig> T fillPasswordFromCacheIfNeed(T connectionConfig) {
```

Comment:
we may change another name "fullFillPasswordIfNeed", and this method may return void~